lib/_stream_writable.js

```diff
@@ -783,44 +734,73 @@ function onFinished(stream, state, cb) {
   stream.prependListener('error', onerror);
 }
 
-ObjectDefineProperty(Writable.prototype, 'destroyed', {
-  // Making it explicit this property is not enumerable
+ObjectDefineProperties(Writable.prototype, {
+
+  // Making it explicit these below properties are not enumerable
   // because otherwise some prototype manipulation in
   // userland will fail
```

Comment:
This comment is no longer necessary?

Reply:
@ronag: Just now pushed these changes, you are too fast ;)